myapp/src/server.js

```diff
@@ -1,3 +1,4 @@
+require("dotenv").config({ path: "../.env" });
```

Comment:
dotenv package is not necessary. The plan is to 
- setup ENV variable outside of Nodejs, and is done in docker stage
- refer to [compose.sh](https://github.com/cwwojin/FORK_backend/blob/main/compose.sh), [docker-compose.yml](https://github.com/cwwojin/FORK_backend/blob/main/docker-compose.yml), [Dockerfile](https://github.com/cwwojin/FORK_backend/blob/main/Dockerfile)